src/pages/about.js

```diff
@@ -4,9 +4,13 @@ import ExecutiveLeaders from '../components/ExecutiveLeaders';
 import Description from '../components/Description';
 import ArchiveCarousel from '../components/ArchiveCarousel';
 import WhoAreWe from '../components/WhoAreWe';
-
+import StaggeredSections from "../components/StaggeredSections/staggeredSections";
+import {exampleObjects} from "../api/exampleStaggeredText/exampleOBJ";
+import {exampleDivs} from "../api/exampleStaggeredText/exampleDIV";
```

Comment:
Even if they are examples, they do need to be in a specific place, so the term example should probably not be prepended here.